notebooks/matrix_tube_pipeline_seqcount_norm.ipynb

```diff
@@ -1584,14 +1575,25 @@
   {
    "metadata": {},
    "cell_type": "markdown",
-   "source": "#### Make and save NovaSeq6000 sample sheet"
+   "source": [
+    "#### Make and save NovaSeq sample \n",
+    "\n",
+    "Run the cell below to view the available NovaSeq sequencer options:"
+   ]
+  },
+  {
+   "metadata": {},
+   "cell_type": "code",
+   "source": "[x for x in REVCOMP_SEQUENCERS + OTHER_SEQUENCERS if x.lower().startswith(\"novaseq\")]",
+   "outputs": [],
+   "execution_count": null
   },
   {
    "metadata": {},
    "cell_type": "code",
    "source": [
     "## INPUT\n",
-    "# Write the novaseq samplesheet as .csv\n",
+    "novaseq_sequencer = 'NovaSeqXPlus'\n",
```

Comment:
@AmandaBirmingham should notebooks also use the string constants we're now defining in metapool or is it more user-friendly for the wetlab to keep using the string literals?